book/history.md

```diff
@@ -214,22 +217,22 @@ worth quoting here (highlight is mine):
 [^enquire]: Enquire was a predecessor web-like database system, also
     written by Berners-Lee.
 
-This quote captures one of the key value propositions of the web. The web was
-successful for several reasons, but I believe it’s primarily the following
-three:
+This quote captures one of the key value propositions of the web: its
+decentralized nature. The web was successful for several reasons, but they
+all had to do with decentralization:
 
- - It provides a very low-friction way to publish information and applications.
-There is no gatekeeper to doing anything, and it’s easy for novices to make a
-simple web page and publish it.
+ - It provided a very low-friction way to publish information and applications.
+   There was no gatekeeper to doing anything, and it was easy for novices to
+   make a simple web page and publish it.
 
- - Once bootstrapped, it builds quickly upon itself via [network
+ - Once bootstrapped, it built quickly upon itself via [network
 effects](https://en.wikipedia.org/wiki/Network_effect) made possible by
 compatibility between sites and the power of the hyperlink to reinforce this
-compatibility. Hyperlinks drive traffic between sites, but also into the web
+compatibility. Hyperlinks drove traffic between sites, but also into the web
 _from the outside_, from sources such as email, social networking, and search
 engines.
```

Comment:
There's still something off to me about these bullet points. I think part of it is the way that the first bullet point is benefit-then-reason, while the second does kind of both at once, while the third is reason-then-benefit. I would rephrase all of them to be reason-then-benefit:

> - Because there was no gatekeeper to doing anything, it was easy for novices to make simple web pages and publish them.
> - Because pages were identified simply by URLs, traffic could come to the web from outside sources like email, social networking, and search engines.
> - Because sites were compatible, the web built quickly upon itself via network effects.
> - Because the web was outside the control of any one entity---and kept that way via standards organizations---it avoided the problems of monopoly control and manipulation.